src/ontology/mondo-edit.obo

```diff
@@ -216712,20 +216712,20 @@ property_value: confidence "0.09890109890109877" xsd:double
 
 [Term]
 id: MONDO:0011071
-name: hereditary thrombocytopenia with normal platelets-hematological cancer predisposition syndrome
-def: "Familial platelet disorder with predisposition to acute myelogenous leukemia (FPD/AML) is characterised by moderate thrombocytopenia, abnormal platelet function and the propensity to develop myeloid malignancies, in particular AML." [Orphanet:71290]
+name: hereditary thrombocytopenia and hematologic cancer predisposition syndrome
+def: "The disorder is characterized by thrombocytopenia of varying severity and a predisposition to hematologic malignancies. It may be caused due to germ line variations in the RUNX1, ETV6 or ANKRD26 genes." [PMID:28600339, https://clinicalgenome.org/affiliation/50034/]
```

Comment:
The def lists 3 genes. However, this declared equivalent to https://omim.org/entry/601399 which has a single causal gene.

If the intent is to group then make this a superclass of a class equiv to the OMIM

Reply:
Oh I see what you've done, this is good, thanks

Reply:
I merged but I just noticed you didn't move the OMM equivalence axiom down - can you make a new ticket for this?